### PR TITLE
merge me_dev to dev

### PR DESCRIPTION
**Pull request by muwyse**
This PR upstreams a handful of minor bug fixes and the BedRock LCE-CCE-MEM interface refactor from me_dev to dev.

### CODE REVIEW

**Comment by dpetrisko** [comment]
Let's hold off until https://github.com/bespoke-silicon-group/basejump_stl/pull/232 to merge.  I pinged for feedback

**Comment by muwyse** [comment]
> Let's hold off until [bespoke-silicon-group/basejump_stl#232](https://github.com/bespoke-silicon-group/basejump_stl/pull/232) to merge. I pinged for feedback

Sounds good. I'm still debugging a few final issues from regressions.

**Comment by muwyse** [comment]
@dpetrisko do you want to do a review pass on this while it is still blocked by https://github.com/bespoke-silicon-group/basejump_stl/pull/232?

All the regressions pass and I think it is good to go otherwise.

**Comment by dpetrisko** [comment]
Ya I’ll take a look today

**Review by dpetrisko — APPROVED** [review]

File: bp_fe/test/tb/bp_fe_icache/wrapper.v
Comment (dpetrisko) [inline bp_fe/test/tb/bp_fe_icache/wrapper.v:49]: Usually we drop the bp_bedrock prefix here since it's a localparam (throughout)

File: bp_fe/test/tb/bp_fe_icache/wrapper.v
Comment (dpetrisko) [inline bp_fe/test/tb/bp_fe_icache/wrapper.v:228]: I think these should be the actual structs

Comment (dpetrisko) [reply bp_fe/test/tb/bp_fe_icache/wrapper.v:228]: (throughout).  It makes it a little easier to debug, since you can grab the struct and inspect it at more levels of the hierarchy

File: bp_fe/test/tb/bp_fe_icache/wrapper.v
Comment (muwyse) [reply bp_fe/test/tb/bp_fe_icache/wrapper.v]: done

File: bp_fe/test/tb/bp_fe_icache/wrapper.v
Comment (muwyse) [reply bp_fe/test/tb/bp_fe_icache/wrapper.v]: fixed (I think I found them all)

**Comment by muwyse** [comment]
@dpetrisko this has been updated to no longer rely on bsg_wormhole_streamer branch in basejump. That will be a second change/PR once the basejump merge to master happens.

I would like to merge after regressions pass unless there are comments/objections.

**Comment by dpetrisko** [comment]
LGTM.  CI is slightly broken as is, I'm working on a patch to fix that. No objections to merging even if the checkpoint jobs hang here

**Comment by dpetrisko** [comment]
Good catch on the clock name thing!

**Pull request merged**
